targets/arduino-mega1280.json

```
@@ -0,0 +1,8 @@
+{
+    "inherits": ["atmega1280"],
+    "build-tags": ["arduino_mega1280"],
+    "ldflags": [
+        "-Wl,--defsym=_bootloader_size=8192"
```

According to the docs it has a bootloader size of 4kB, so this should be:

```suggestion
        "-Wl,--defsym=_bootloader_size=4096"
```

The boards.txt file says the same thing (128kB - 126976 = 4096):

```
mega.menu.cpu.atmega1280.upload.maximum_size=126976
```